Makefile

```diff
@@ -198,6 +204,11 @@ controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessar
 $(CONTROLLER_GEN): $(LOCALBIN)
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
 
+.PHONY: crdocs
+crdocs: $(CRDOC) ## Download crdoc locally if necessary.
+$(CRDOC): $(LOCALBIN)
+	GOBIN=$(LOCALBIN) go install fybrik.io/crdoc@latest
```

Comment:
Should we pin it and let renovate handling updates?

Reply:
I will add the current version as an env var, but we should setup renovate separately since[ it looks like](https://github.com/open-feature/open-feature-operator/issues/153) it is not handling make files

Reply:
#450